lib/tasks/export.thor

```diff
@@ -0,0 +1,45 @@
+require 'thor'
+require './config/environment' # Load Rails
+
+class Export < Thor
+  desc 'children FILE', 'Exports children from the database to FILE (defaults to tmp/all.csv)'
+  method_option :include, default: 'complete', enum: %w[complete incomplete all], aliases: '-i', desc: 'Which applications to include'
+  method_option :format, default: 'csv', enum: %w[csv], aliases: '-f', desc: 'Which format to output'
+  method_option :after, aliases: '-a', desc: 'Export children submitted after BEGIN. Does not work with incomplete/all.'
+  method_option :before, aliases: '-b', desc: 'Export children submitted before END. Does not work with incomplete/all.'
+  method_option :today, type: :boolean, desc: 'Export children submitted today. Does not work with incomplete/all. Overwrites after/before.'
+  method_option :yesterday, type: :boolean, desc: 'Export children submitted yesterday. Does not work with incomplete/all. Overwrites after/before.'
+  method_option :count, type: :boolean, aliases: '-c', desc: 'Only counts the number that would be exported, but does not actually export anything.'
+  method_option :hours_ago, type: :numeric, aliases: '-h', desc: 'Export children submitted after HOURS_AGO hours ago. Does not work with incomplete/all. Overwrites after.'
```

Comment:
This isn't part of the acceptance criteria, and although it could be nice to have we should hold off on implementing behavior that product hasn't asked for to not unintentionally expose ourselves to bugs.

Reply:
Slimed this down to purely start/end date